tests/e2e/performance/test_pvc_multi_snapshot_performance.py

```diff
@@ -94,65 +98,20 @@ def test_pvc_multiple_snapshot_performance(
-            log.info(
-                f'Running IO on pod {self.pod_obj.name} - Test num. {test_num}'
-            )
-            self.pod_obj.fillup_fs(size=file_size, fio_filename=file_name)
-
-            # Wait for fio to finish
-            fio_result = self.pod_obj.get_fio_results()
-            err_count = fio_result.get('jobs')[0].get('error')
-            assert err_count == 0, (
-                f"IO error on pod {self.pod_obj.name}. "
-                f"FIO result: {fio_result}"
-            )
-            log.info('IO on the PVC Finished')
-
-            # Take a snapshot of the PVC and measure the time of creation.
-            snap_name = self.pvc_obj.name.replace(
-                'pvc-test', f'snapshot-test{test_num}'
-            )
-            log.info(f'Taking snapshot of the PVC {snap_name}')
-            snap_obj = pvc.create_pvc_snapshot(
-                self.pvc_obj.name,
-                snap_yaml,
-                snap_name,
-                helpers.default_volumesnapshotclass(self.interface).name,
-            )
-            snap_obj.ocp.wait_for_resource(
-                condition='true', resource_name=snap_obj.name,
-                column=constants.STATUS_READYTOUSE, timeout=60
-            )
-
-            snap_con_name = snap_obj.ocp.get(
-                resource_name=snap_name,
-                out_yaml_format=True
-            )["status"]["boundVolumeSnapshotContentName"]
-            log.info(f'Snap content name in test-{test_num} is {snap_con_name}')
-
-            test_results['create']['time'] = helpers.measure_snapshot_creation_time(
-                self.interface, snap_obj.name, snap_con_name
-            )
-            test_results['create']['speed'] = int(
-                filesize * constants.GB2MB / test_results['create']['time']
-            )
-            log.info(f'Test {test_num} results:')
-            log.info(f'Snapshot creation time is : {test_results["create"]["time"]} sec.')
-            log.info(f'Snapshot speed is : {test_results["create"]["speed"]} MB/sec')
-            create_times.append(test_results)
-
-        logging.info(f"Snapshot created results are:  {create_times}")
+        os.environ["SNAPNUM"] = f'{num_of_snaps}'
+        os.environ["LOGPATH"] = f'{ocsci_log_path()}'
+        os.environ["FILESIZE"] = file_size
+        os.environ["NSPACE"] = self.pvc_obj.namespace
+        os.environ["PODNAME"] = self.pod_obj.name
+        os.environ["PVCNAME"] = self.pvc_obj.name
+        os.environ["INTERFACE"] = self.interface
+
+        main_script = "tests/e2e/performance/test_multi_snapshots.py"
+        result = subprocess.run([main_script], stdout=subprocess.PIPE)
+        log.info(f"Results from main script : {result.stdout.decode('utf-8')}")
+
+        if 'All results are' not in result.stdout.decode('utf-8'):
+            log.error('Test did not compleated')
+            raise Exception('Test did not compleated')
```

Comment:
```suggestion
           'Test did not complete'
```

Reply:
if you meant the typo, i fixed it, if other thing, i did not understand.